utils/lint_test_cases.sh

```diff
@@ -1,28 +1,7 @@
 #! /bin/bash
 
-for f in `find ./test-cases/ -type f \\( -iname "*.yaml" ! -iname "*multi-dc.yaml" ! -iname *multiDC*.yaml ! -iname *multiple-dc*.yaml ! -iname *rolling* ! -iregex .*docker.* ! -iregex .*artifacts.* !  -iregex .*private-repo.* ! -iregex .*ics/long.* ! -iregex .*scylla-operator/.* ! -iregex .*gce.* ! -iregex .*jepsen.* \\)` ; do
-    echo "---- linting: $f -----"
-    RES=$( script --flush --quiet --return /tmp/test-case.txt --command "SCT_SCYLLA_VERSION=4.0.0 python3 ./sct.py conf $f" )
-    if [[ "$?" == "1" ]]; then
-        cat /tmp/test-case.txt
-        exit 1;
-    fi
-done
+SCT_SCYLLA_VERSION=4.2.0 ./sct.py lint-yamls -i '.yaml' -e 'multi-dc.yaml,multiDC,multidc,multiple-dc,rolling,docker,artifacts,private-repo,ics/long,scylla-operator,gce,jepsen'
```

Comment:
Why 4.2.0?

Reply:
It is better to put something fresh like 4.4

Reply:
done